Sources/UIApplication+handleSDLEvents.swift

```diff
@@ -228,34 +229,33 @@ import JNI
 public func onNativeTouch(
     env: UnsafeMutablePointer<JNIEnv?>?,
     view: JavaObject?,
-    touchDeviceID: JavaInt,
-    pointerFingerID: JavaInt,
-    action: JavaInt,
-    x: JavaFloat,
-    y: JavaFloat,
-    pressure: JavaFloat,
-    timestampMs: JavaLong
+    touchParameters: JavaObject
 ) {
     if env == nil || view == nil {
         // We're in an invalid state where the JNI has exploded somehow
         // Passing anything on to UIKit now would probably lead to a crash
         return
     }
 
+    let action: JavaInt = try! jni.GetField("action", from: touchParameters)
+    let x: JavaFloat = try! jni.GetField("x", from: touchParameters)
+    let y: JavaFloat = try! jni.GetField("y", from: touchParameters)
+    let timestampMs: JavaLong = try! jni.GetField("timestamp", from: touchParameters)
+
     guard let eventType = SDL_EventType.eventFrom(androidAction: action)
     else { return }
 
     var event = SDL_Event(tfinger:
         SDL_TouchFingerEvent(
             type: eventType.rawValue,
             timestamp: UInt32(timestampMs),
-            touchId: Int64(touchDeviceID), // some arbitrary number, stays the same per device
-            fingerId: Int64(pointerFingerID),
+            touchId: 0,
+            fingerId: 0,
             x: x / Float(UIScreen.lastKnownScale),
             y: y / Float(UIScreen.lastKnownScale),
             dx: 0,
             dy: 0,
-            pressure: pressure
+            pressure: 0
```

Comment:
still omitted the unused props. See comment https://github.com/flowkey/UIKit-cross-platform/pull/368#issuecomment-1152423925